_data/changelogs/docs-settings.yml

```
@@ -2,6 +2,13 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added color contrast checks for disabled tokens.
+    summaryAdditional:  On compilation, USWDS will test disabled element color contrast and provide a fallback color if minimum contrast is not met. If the fallback does not meet minimum requirements, a warning will be provided.
+    affectsStyles: true
```

I feel like this might qualify as an accessibility item 🤔 What do you think?

```suggestion
    affectsAccessibility: true
    affectsStyles: true
```
```

---

I like it! I'll commit your suggestion